slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -128,9 +144,8 @@ void SLAMServiceImpl::CreateMap() {
 
     if (this->starting_scan_number < 0 ||
```

Comment:
Are we using `starting_scan_number`, or should we remove this and always start from zero?

Reply:
I think we should keep this for now, since we'll later connect this functionality once we implement `"input_file_pattern": "filepathglob"` (which is why we have it to begin with in the tech spec)

Reply:
I am not sure. I think it's just not high enough priority right now, but I'd say the debate about it is still to be discussed. It's in the tech spec 🤷🏻‍♀️ And when we were testing, it was a helpful debugging tool. I can imagine it being useful to implement - especially for cartographer

Reply:
Sounds good! Added to the agenda for the SLAM meeting.